src/cpp/shared_core/system/encryption/EncryptionVersionTests.cpp

```diff
@@ -80,225 +80,225 @@ bool decryptedPayloadMatches(std::vector<unsigned char>& decryptedData)
    return g_payload == decryptedPayload;
 }
 
-test_context("EncryptionVersionTests")
+TEST(SharedCoreTest, EncryptionVersionTests)
 {
-   test_that("v0: Can AES encrypt/decrypt")
```

Comment:
this eliminates this "test_that" function, but there still are curlys to handle scope... I think that works?